advocacy_docs/partner_docs/DBeaverGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring DBeaver'
+---
+
+Implementing DBeaver with EDB Postgres Advanced Server and EDB Postgres Extended requires the following components: 
+
+- A running EDB Postgres Advanced Server or EDB Postgres Extended instance.
+- DBeaver application installed on a system where you will manage the running database.
+
+## EDB Postgres Advanced Server Configuration
+
+1. Launch DBeaver on your system. 
+2. Go to the `Database` drop down menu in the top left, and select `New Database Connection`. In the `All` tab, select `EnterpriseDB` and click `Next`.    
+
+<p align="center">
+<img src="Images/NewDatabaseConnection.png">
+</p>
+
+
+!!! Note
+    You can also connect to your EDB database instance by selecting `New Connection from JDBC URL` from the `Database` menu. DBeaver will automatically select EnterpriseDB in the `Drivers` field when the EDB JDBC URL is provided in the `JDBC URL` field. Click `Proceed` and continue to step 3.
+
+
+
+<p align="center">
+<img src="Images/JDBCURL.png">
+</p>
+
+3. Provide connection information for the user, password, host, port and database for your EDB Postgres Advanced Server instance.
+<p align="center">
+<img src="Images/ConnectionSettings.png">
+</p>
+
+4. Click `Test Connection`. The connection should be successful. 
+<p align="center">
+<img src="Images/SuccessfulConnection.png">
+</p>
+
+5. Click `Ok` on this window and then click `Finish` on the `Connection Settings` window. The connection is then created and shown in the left-side tree.  
+<p align="center">
+<img src="Images/ConnectionOnLeftSide.png">
+</p>
+
+## EDB Postgres Extended Configuration
+
+1. Launch DBeaver on your system. 
+2. Go to the `Database` drop down menu in the top left, and select `New Database Connection`. In the `All` tab, select `PostgreSQL` and click `Next`.
+<p align="center">
+<img src="Images/ExtendedConfiguration.png">
+</p>
+
```

Comment:
you can see the spacing for extended between image - note - image is good.  Just pointing it out as an example to see.  Not edit needed on this part.